.github/workflows/push.yml

```diff
@@ -31,8 +31,8 @@ jobs:
       - name: Build
         run: yarn build
       # TODO: Enable tests before v7 release
```

Comment:
Remove `TODO` comment, otherwise tests seem to pass so it looks this can be merged?